package.json

```diff
@@ -43,7 +43,7 @@
     "docker": "yarn docker:artifacts && yarn docker:deployer && yarn docker:node",
     "docker:artifacts": "docker rmi --force api3/artifacts && docker build --tag api3/artifacts:latest --file docker/Dockerfile .",
     "docker:deployer": "docker build --tag api3/deployer:latest --file packages/deployer/docker/Dockerfile .",
-    "docker:node": "docker build --tag api3/airnode:latest --file packages/node/docker/Dockerfile .",
+    "docker:node": "docker build --tag api3/airnode-client:latest --file packages/node/docker/Dockerfile .",
```

Comment:
Because the image is called like this in docker hub https://hub.docker.com/r/api3/airnode-client

@bbenligiray @amarthadan do we want to use the same name as in pre-alpha for the dockers (same question for deployer).

Reply:
I personally like `api3/airnode-deployer` and `api3/airnode-client` as image names/have been using them